geosys/test/mock/geosys_api_server_app.py

```diff
@@ -134,10 +134,10 @@ def management_zones_map(map_type):
         return jsonify({"error": "Invalid headers"}), 400
 
     data = request.get_json()
-    if not data or "SeasonField" not in data or "Images" not in data or "params" not in data:
+    if not data or "SeasonField" not in data or "Images" not in data:
         return jsonify({"error": "Invalid request data"}), 400
 
-    zone_count = data.get('params').get("zoneCount")
+    zone_count = data.get("zoneCount")
```

Comment:
is this for the all zones maps in version 5  or only for SAMZ maps?

Reply:
For all. `zoneCount` is no longer a parameter as in `v4`. It is part of the payload.